react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { memo, FC, SyntheticEvent } from 'react'
-import { FormattedNumber } from 'react-intl'
 import classNames from 'classnames'
+import { FormattedNumber } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
 
 import styles from '../styles.css'
```

Comment:
I see that this is still being used at the bottom of the file. Why is that?
Are `skuSelectorItemImageValue` and `skuSelectorBadge` special cases?